tests/platform/cloud/CloudStatusTests.js

```diff
@@ -1,5 +1,5 @@
 /*!
-Copyright 2018 OCAD university
+Copyright 2018, 2019 OCAD university
```

Comment:
The format for years is usually "2018-2019".

Reply:
There are rules regarding use of comma vs. dash in copyright dates.  In this particular case, the use of either is legitimate and amounts to the same meaning.  I'll use the dash.